### PR TITLE
Handle small image for content moderator

### LINKED ISSUE #2191: Content Moderator: Handle Small Image
## Summary

When a user uploads an image as evidence the application calls Azure's Content Moderator service to analyze the file. According to Microsoft documentation the file size is limited to **minimum 128 pixels, maximum 4MB size**. Unfortunately this is not accurate as the limit of a minimum 128 pixels applies to length and width not the total number of pixels.  This is resulting in an error when the user uploads a small image (i.e. 400 pixels x 125 pixels). 

Currently, the application does not handle this error and the evidence page does not inform users about this limitation. We need to update the page to notify the user, and display an error message when Content Moderator rejects the file. Error handling may be needed in scanFiles.js as well.

### PR DESCRIPTION
**Pull request by ying-chen-cds**
Fixes #2191

# Description

> In "Attach supporting evidence" page, if you upload an image with the width or length less than 128 pixel (which is not accepted by content moderator), there will be an alert popup : "Warning: Your image is too small(either the width or the height is less than 128 pixels). Please increase the image size  and try uploading again."

# Any new packages installed?

> Give details

# Required followup work

> Is there anything related to this that still needs to be done (ex: documentation changes).

# Checklist:

- [ ] I have updated the azurescript.sh with any **new environment variables** and added them to the appsettings
- [ ] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [ ] I have added and needed tests for my changes (in particular for new screens)
- [ ] I have added a comment to any confusing code
- [ ] I have added documentation to any modified front-end code. (Or added missing documentation)

### CODE REVIEW

**Comment by justinr86** [comment]
I have a PR in to refactor this form. Please wait until that PR is approved and merged.

**Comment by ying-chen-cds** [comment]
> I have a PR in to refactor this form. Please wait until that PR is approved and merged.

Yes, I've noticed. Thanks!

File: f2/src/forms/EvidenceInfoForm.js
Comment (justinr86) [inline f2/src/forms/EvidenceInfoForm.js:75]: Now that the refactored form has been merged we should use the modal to display error messages here instead of alert.

**Review by sdzhangtao — APPROVED** [review]

**Pull request merged**
